scene/3d/collision_polygon_3d.cpp

```diff
@@ -155,15 +155,23 @@ bool CollisionPolygon3D::is_disabled() const {
 }
 
 String CollisionPolygon3D::get_configuration_warning() const {
+	String warning = Node3D::get_configuration_warning();
+
 	if (!Object::cast_to<CollisionObject3D>(get_parent())) {
-		return TTR("CollisionPolygon3D only serves to provide a collision shape to a CollisionObject3D derived node. Please only use it as a child of Area3D, StaticBody3D, RigidBody3D, KinematicBody3D, etc. to give them a shape.");
+		if (!warning.empty()) {
+			warning += "\n\n";
+		}
+		return warning + TTR("CollisionPolygon3D only serves to provide a collision shape to a CollisionObject3D derived node. Please only use it as a child of Area3D, StaticBody3D, RigidBody3D, KinematicBody3D, etc. to give them a shape.");
```

Comment:
Same here, not doing an early return should provide both warnings to the user if both conditions are true, which makes sense as both need to be fixed.